packages/contact-center/user-state/tests/helper.ts

```diff
@@ -0,0 +1,90 @@
+import { renderHook, act, waitFor } from '@testing-library/react';
+import { useUserState } from '../src/helper';
+
+describe('useUserState Hook', () => {
+  const mockCC = {
+    setAgentState: jest.fn()
+  };
+
+  const idleCodes = [
+    { id: '1', name: 'Idle Code 1', isSystem: false },
+    { id: '2', name: 'Available', isSystem: false }
+  ];
+
+  const agentId = 'agent123';
+
+  beforeEach(() => {
+    jest.useFakeTimers();
+    mockCC.setAgentState.mockReset();
+  });
+
+  afterEach(() => {
+    jest.useRealTimers();
+  });
+
+  it('should initialize with default values', () => {
+    const { result } = renderHook(() => useUserState({ idleCodes, agentId, cc: mockCC }));
+
+    expect(result.current.isSettingAgentStatus).toBe(false);
```

Comment:
Instead of checking each prop one by one, we can just add complete prop check at once

Reply:
Done